config.reek

```diff
@@ -20,6 +20,13 @@ LongParameterList:
   exclude:
   - initialize
 
+UncommunicativeMethodName:
+  accept:
+    - s3 # elb, rds, ec2, sqs are all fine
+UncommunicativeVariableName:
+  accept:
+    - s3 # elb, rds, ec2, sqs are all fine
```

Comment:
We probably should add those, too. But there's probably dozens of AWS services with numbers, and I only added the one that was bothering me. `ec2` is probably a good one to add, do kale or arugula use any others?